README.md

```diff
@@ -7,57 +7,82 @@
 
+<br/>
+<p align="center">
+    <img width="80%" src="https://sulu.io/uploads/media/800x@2x/02/342-SuluRedirectBundle%20Slideshow.gif?v=1-0" alt="SuluRedirectBundle Slideshow">
 </p>
+<br/>
+
+The SuluRedirectBundle is compatible with Sulu **starting from version 2.0**. Have a look at the `require` section in
+the [composer.json](https://github.com/sulu/SuluRedirectBundle/blob/2.x/composer.json) to find an
+**up-to-date list of the requirements** of the bundle.
+
+## 🚀&nbsp; Installation and Documentation
+
+Execute the following [composer](https://getcomposer.org/) commands to add the bundle to the dependencies of your
+project:
 
-The SuluRedirectBundle adds support for managing redirects in Sulu.
+```bash
+composer require sulu/redirect-bundle
+```
 
-Additional features included:
+Afterwards, visit the [bundle documentation](https://github.com/sulu/SuluRedirectBundle/blob/2.x/Resources/doc) to
+find out **how to set up and configure the SuluRedirectBundle** to your specific needs.
 
-* Content-Manager can create new redirect-routes in Sulu-Admin
-* Import redirect-routes from csv
+## 💡&nbsp; Features
 
-## Status
+### Importing redirects
 
-This repository will become version 1.0 of SuluRedirectBundle. It is under **heavy development** and currently its APIs
-and code are not stable yet (pre 1.0).
+One of the great features of this bundle is the ability to import redirects from a CSV file.
+The most simplified file just contains two columns, `source` and `target`.
+Of course, all the other options like `statusCode`, `sourceHost`, and `enabled` can also be set in the import file.
 
-## Requirements
+It’s also possible to override existing redirects with an import; you just have to set the same value for `source`.
 
-* Composer
-* PHP `^7.2`
-* Sulu `^2.0.0`
+### Enabling and disabling redirects
 
-For detailed requirements see [composer.json](https://github.com/sulu/SuluRedirectBundle/blob/2.x/composer.json).
+Sometimes it’s necessary to prepare redirects which are not ready yet, especially if you have a large number of redirects.
+Of course there’s a solution for that — toggler in the toolbar to enable or disable redirects quickly.
+This allows you to import a large number of disabled redirects and then check them in the administration interface,
+before enabling them when needed.
 
-## Documentation
+### Different statuses
 
-The the Documentation is stored in the
-[Resources/doc/](https://github.com/sulu/SuluRedirectBundle/blob/2.x/Resources/doc) folder.
+The SuluRedirectBundle comes with three different statuses to be used for redirects:
 
-## Installation
+- **301** Moved permanently
+- **302** Moved temporarily
+- **410** Gone
 
-All the installation instructions are located in the
-[Documentation](https://github.com/sulu/SuluRedirectBundle/blob/2.x/Resources/doc/installation.md).
+Each of them is useful in different situations, but I think they are quite self explanatory.
```

Comment:
I think this sentence is not needed and we should avoid using personal opinions in a readme. But we could add the links to explain the status codes e.g. https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/301

Reply:
oops didn't saw the comment from @nnatter 😅